js/clinicalTimeline.js

```diff
@@ -169,15 +169,37 @@ var clinicalTimeline = (function(){
       .attr("width", overviewAxisWidth)
       .attr("class", "overview");
 
+    clinicalTimelineOverviewAxis(overviewSVG, getTickValues, minDays, maxDays, overviewAxisWidth, formatTime, daysToTimeObject);
+    clinicalTimelineVerticalLine(timelineBool.enableVerticalLine, beginning, ending, timelineBool.advancedView, timelineBool.tooltipOnVerticalLine);
```

Comment:
This PR still requires one to add two lines to `timeline` i.e. `clinicalTimelineOverviewAxis()`and `clinicalTimelineVerticalLine()`. They also both have their own function arguments. As explained in #95 I think it would be more clean if all plugins exposed the same public function `run(timeline)` so it is clear for anybody developing a new plugin, that's the type of function they should expose, which is then subsequently called here in clinicalTimeline. Ideally this would just be some for loop that calls `run(timeline)` on each enabled plugin. You should probably also add some function that makes sure no plugin conflicts appear. That is you can't combine the zoom plugin with the trimmed timeline plugin. You could handle this inside the plugin. Making sure no other plugin that conflicts is enabled. Another way would be adding a separate file that does that, maybe some `pluginManager` object.